Sources/Sentry/SentryEnvelope.m

```diff
@@ -126,45 +126,11 @@ - (instancetype)initWithEvent:(SentryEvent *)event
     }
 
     return [self
-        initWithHeader:[[SentryEnvelopeItemHeader alloc] initWithType:SentryEnvelopeItemTypeEvent
+        initWithHeader:[[SentryEnvelopeItemHeader alloc] initWithType:event.type
```

Comment:
Wondering whether this change can cause some errors.

Reply:
@philipphofmann